utils/api/benefits/checkGis.ts

```diff
@@ -7,6 +7,8 @@ import {
 import { GisSchema } from '../definitions/schemas'
 import { BenefitResult, CalculationInput } from '../definitions/types'
 import { validateRequestForBenefit } from '../helpers/validator'
+import { OutputItem } from '../scrapers/_base'
+import gisTables from '../scrapers/output'
```

Comment:
interesting, the source of truth for GIS resides in tables on the web somewhere?